elf/module.go

```diff
@@ -143,24 +144,48 @@ type TracepointProgram struct {
 	efd   int
 }
 
+// CustomContext represents a custom context
+type CustomContext struct {
+	Name       string
+	CgroupPath string
```

Comment:
It should not be specific to cgroups, but follow similar pattern to `/etc/iproute2/bpf_pinning`, like a `map[int]string`.